src/routes/docs/products/sites/quick-start/tanstack-start/+page.markdoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: article
+title: Deploy a Tanstack Start app to Appwrite Sites
+description: Learn how to setup and deploy Tanstack Start apps on Appwrite Sites.
+---
+
+{% section #step-1 step=1 title="Create Tanstack Start app" %}
+
+First, you must either create a Tanstack Start app or setup the [Tanstack Start starter template](https://github.com/appwrite/templates-for-sites).
+
+Open your terminal, and run the following command.
+
+```bash
+npm create @tanstack/start@latest
+```
+
+Push this project to a [GitHub repo](https://github.com/new).
+{% /section %}
+
+{% section #step-2 step=2 title="Create Appwrite project" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io).
+
+{% only_dark %}
+![Create Appwrite project](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create Appwrite project](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+If this is your first time using Appwrite, create an account and create your first project.
+{% /section %}
+
+{% section #step-3 step=3 title="Create site" %}
+Head to the **Sites** page in your Appwrite project, click on the **Create site** button, and select **Connect a repository**.
+
+Connect your GitHub account and select the repository you intend to deploy.
+
+1. Select the **production branch** and **root directory** from your repo.
+2. Verify that the **correct framework** is selected. In case an incorrect framework is visible, pick **Tanstack Start** from the drop-down list. 
+3. Confirm the **install command**, **build command**, and **output directory** in the build settings. The default build settings for Tanstack Start are:
+    - **Install command:** `npm install`
+    - **Build command:** `npm run build`
+    - **Output directory:** `./dist`
+4. Add any **environment variables** required by the site.
```

Comment:
No env vars for new tanstack project, we can remove this to make quickstart simpler